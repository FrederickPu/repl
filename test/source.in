{"cmd": "def f := 37"}

{"cmd": "def g := 38"}

{"src": 1}

{"src": 0}

{"cmd": "def h := 39", "env": 0}

{"src": 2}

