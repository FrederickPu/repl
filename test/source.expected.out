{"env": 0}

{"env": 1}

{"src": "def f := 37\ndef g := 38\n"}

{"src": "def f := 37\n"}

{"env": 2}

{"src": "def f := 37\ndef h := 39\n"}

